nilearn/maskers/multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,259 @@
+    NiftiLabelsMasker).
+
+    Parameters
+    ----------
+    labels_img: Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        Region definitions, as one image of labels.
+
+    labels : list of str, optional
+        Full labels corresponding to the labels image. This is used
+        to improve reporting quality if provided.
+        Warning: The labels must be consistent with the label
+        values provided through `labels_img`.
+
+    background_label: number, optional
+        Label used in labels_img to represent background.
+
+    mask_img: Niimg-like object, optional
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        Mask to apply to regions before extracting signals.
+
+    %(smoothing_fwhm)s
+        If smoothing_fwhm is not None, it gives the full-width-at-half-maximum
+        in millimeters of the spatial smoothing to apply to the signal.
+
+    standardize: {'zscore', 'psc', True, False}, default is 'zscore'
+        Strategy to standardize the signal.
+        'zscore': the signal is z-scored. Timeseries are shifted
+        to zero mean and scaled to unit variance.
+        'psc':  Timeseries are shifted to zero mean value and scaled
+        to percent signal change (as compared to original mean signal).
+        True : the signal is z-scored. Timeseries are shifted
+        to zero mean and scaled to unit variance.
+        False : Do not standardize the data.
+
+    standardize_confounds: boolean, optional,  default is True
+        If standardize_confounds is True, the confounds are z-scored:
+        their mean is put to 0 and their variance to 1 in the time dimension
+
+    high_variance_confounds : boolean, optional
+        If True, high variance confounds are computed on provided image with
+        :func:`nilearn.image.high_variance_confounds` and default parameters
+        and regressed out. Default=False.
+
+    detrend: boolean, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    low_pass: None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    high_pass: None or float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    t_r: float, optional
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    dtype: {dtype, "auto"}
+        Data type toward which the data should be converted. If "auto", the
+        data will be converted to int32 if dtype is discrete and float32 if it
+        is continuous.
+
+    resampling_target: {"data", "labels", None}, optional.
+        Gives which image gives the final shape/size. For example, if
+        `resampling_target` is "data", the atlas is resampled to the
```

Comment:
I think a bullet point list for the options would also be more readable here. WDYT?